src/main/java/com/sap/oss/phosphor/fosstars/advice/oss/OssAdviceContentYamlStorage.java

```diff
@@ -34,8 +34,12 @@
 
   static {
     try {
+      // FIXME (mibo): check if this could be done for all in OssFeatures defined
+      //                features instead of being based on a rating
       DEFAULT = OssAdviceContentYamlStorage.loadFromResource(
-          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssSecurityRating.class));
+          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssArtifactSecurityRating.class));
+      //      DEFAULT = OssAdviceContentYamlStorage.loadFromResource(
+      //          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssSecurityRating.class));
```

Comment:
Could you please open an issue instead of adding FIXMEs? It is much easier to track problems/enhancements with issues.

Reply:
Done -> https://github.com/SAP/fosstars-rating-core/issues/474